inbox/push-priority.xml

```diff
@@ -68,19 +69,18 @@
 ]]></example>
 </section1>
 <section1 topic='Enabling Push Notifications Priorities' anchor='enable'>
-  <p>To enable this feature XMPP client SHOULD send enable request with 'priority' element qualified by the 'urn:push:priority:0' namespace:</p>
+  <p>To enable this feature XMPP client sends IQ-set with an <enable /> element containing <priority /> element qualified by the 'urn:push:priority:0' namespace:</p>
   <example caption='Enabling feature'><![CDATA[
 <iq type='set' id='x42'>
-  <enable xmlns='urn:xmpp:push:0' jid='push-5.client.example'
-    node='yxs32uqsflafdk3iuqo' >
+  <enable xmlns='urn:xmpp:push:0' jid='push-5.client.example' node='yxs32uqsflafdk3iuqo' >
     <priority xmlns='urn:xmpp:push:priority:0' />
   </enable>
 </iq>    
 ]]></example>
-  <p class='em'>Note: The XMPP client SHOULD enable this feature ONLY if the XMPP server supports all the required push notifications filtering.</p>
+  <p class='em'>Note: The XMPP client SHOULD enable this feature ONLY if the XMPP server supports all required push notifications filtering.</p>
 </section1>
 <section1 topic='Marking push notification with priority' anchor='marking'>
-  <p>If this feature is enabled and high priority push notification is being sent, the XMPP server MUST mark it by adding 'priority' element qualified with 'urn:xmpp:push:priority:0' namespace and value set to high to the 'notification' element of the push notification before publishing it.</p>
+  <p>If this feature is enabled and high priority push notification is being sent, the XMPP Server MUST indicate this by adding to the <notification /> element a <priority /> element qualified by the 'urn:xmpp:push:priority:0' namespace with it's value set to 'high':</p>
```

Comment:
Why you replaced "'" with "<" and ">"? This document is XML so you would need to escape those characters or they will be "replaced" by renderer with some unknown behaviour. In examples that is OK (they are in CDATA) but not in here! (at least that is my impression.

Reply:
Why? Again - I was basing it on other XEPs, see https://xmpp.org/extensions/xep-0357.html#publishing. But you are correct that those have to be escaped.

Reply:
I was only referring to escaping those values or reverting those changes. That was all.